docs/testing.md

```diff
@@ -16,10 +16,10 @@ By default snapshots with emotion show generated class names. Adding [jest-emoti
 npm install --save-dev jest-emotion
 ```
 
-Add the snapshot serializer in your [`setupTestFrameworkScriptFile`](http://facebook.github.io/jest/docs/en/configuration.html#setuptestframeworkscriptfile-string) _or_ at the top of your test file.
+Add the snapshot serializer in your [`setupFilesAfterEnv`](https://jestjs.io/docs/en/configuration.html#setupfilesafterenv-array) _or_ at the top of your test file.
 
 ```javascript
-import * as emotion from 'emotion'
+import * as emotion from '@emotion/core'
 import { createSerializer } from 'jest-emotion'
 
 expect.addSnapshotSerializer(createSerializer(emotion))
```

Comment:
Thanks for the PR, it looks like this doc wasn't updated when some changes to jest-emotion were made and this is quite inaccurate. Could you change this section to be more like what's at https://github.com/emotion-js/emotion/tree/master/packages/jest-emotion#snapshot-serializer?

Reply:
Not sure what kind of a change is desired here - should we maybe just duplicate the intro from the linked docs?